data-plane/dispatcher-loom/pom.xml

```diff
@@ -30,6 +30,7 @@
 
   <properties>
     <java.version>20</java.version>
+    <argLine>--enable-preview</argLine>
```

Comment:
why is this needed?

Reply:
As we are using the Java Project Loom virtual threads. which is still a preview feature in JDK20.
It will be stable in JDK21 which will release this sep I believe.

Reply:
Sure